radio/src/pulses/pulses_arm.cpp

```diff
@@ -209,6 +209,8 @@ void setupPulses(uint8_t port)
 #endif
     case PROTO_PPM:
       setupPulsesPPM(port);
+      // 45 ms is the worst case length of the PPM frame length (800 µs between channels and 16 channels
```

Comment:
This is not the reason of the 45!
22.5ms is the default length of a PPM frame, it's stored in EEPROM as "0" to have a better compression ratio.